content/home/marketing/component-based.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Component-Based
+title: Berbasis Komponen
 order: 1
 ---
 
-Build encapsulated components that manage their own state, then compose them to make complex UIs.
+Bangun komponen-komponen terenkapsulasi yang mengatur *state*-nya sendiri, kemudian gabungkan mereka untuk membentuk UI yang lebih kompleks.
 
-Since component logic is written in JavaScript instead of templates, you can easily pass rich data through your app and keep state out of the DOM.
+Karena logika pada komponen React ditulis langsung menggunakan JavaScript (bukan menggunakan *template*), anda dapat dengan mudah mengoper data melalui aplikasi anda dan menempatkan *state* agar tetap berada di luar DOM.
```

Comment:
Kata _template_ bisa digantikan dengan kata **templat**. Mungkin tidak terkenal, tetapi maknanya tetap sama

https://kbbi.kemdikbud.go.id/entri/templat

Kemudian, kata "anda" seharusnya dituliskan kapital "Anda" karena berupa sapaan kepada pembaca.

Reply:
> Kemudian, kata "anda" seharusnya dituliskan kapital "Anda" karena berupa sapaan kepada pembaca.

Hi, is there any reference about kata sapaan non-gelar that had to be capitalized? Tried to find it in PUEBI but didn't find any https://puebi.readthedocs.io/en/latest/huruf/huruf-kapital/